townService/src/town/games/ChessGame.ts

```diff
@@ -35,6 +35,8 @@ export default class ChessGame extends Game<ChessGameState, ChessMove> {
       moves: [],
       status: 'WAITING_TO_START',
       halfMoves: 0,
+      capturedBlackPieces: [],
+      capturedWhitePieces: [],
```

Comment:
I assume this will also error, I only really found after trying to start the backend locally